### ISSUE TITLE
t/db.t: plan error

### ISSUE THREAD

**Issue by eserte** [RexOps/Rex#714]
Running the test suite of Rex-1.3.0 may fail:
```
You tried to plan twice at t/db.t line 26.
# Looks like your test exited with 2 before it could output anything.
t/db.t ........................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 38/38 subtests 
```

**Comment by ferki** [comment]
@eserte: thanks for the report! Looks like this slipped through our current test suite :/

**Comment by ferki** [comment]
@eserte: I guess this is the original [CPAN Testers report](http://www.cpantesters.org/cpan/report/144ddb70-0d18-11e5-b140-97c7e0bfc7aa), right?

**Comment by eserte** [comment]
> I guess this is the original CPAN Testers report, right?

Yes.

**Comment by ferki** [comment]
@eserte: Thanks for your feedback! This should be fixed in 1.3.1, which is already on CPAN and in our package repositories. Proper announcement is expected soon.
